src/material-experimental/theming/_m3-tokens.scss

```diff
@@ -234,7 +401,10 @@
     ),
     _namespace-tokens(
       (mdc, outlined-text-field),
-      mdc-tokens.md-comp-outlined-text-field-values($systems, $exclude-hardcoded),
+      _fix-outlined-text-field-tokens(
+        mdc-tokens.md-comp-outlined-text-field-values($systems, $exclude-hardcoded),
+        mdc-tokens.md-comp-outlined-text-field-values($systems, false),
```

Comment:
Context here: the opacity values are only included when hardcoded values are enabled, but because we only use the opacity for calculations, we pass in the tokens with hardcoded values separately.

Reply:
This sounds like it would have been a good comment, but happy to leave this to you

Reply:
Added a comment.